consensus/obcpbft/pbft-core.go

```diff
@@ -192,6 +192,13 @@ func newPbftCore(id uint64, config *viper.Viper, consumer innerStack) *pbftCore
 	instance.idleChan = make(chan struct{})
 	instance.injectChan = make(chan func())
 
+	// TODO Ultimately, the timer factory will be passed in, and the existence of the manager
+	// will be hidden from pbftCore, but in the interest of a small PR, leaving it here for now
+	instance.manager = newEventManagerImpl(instance)
+	etf := newEventTimerFactoryImpl(instance.manager)
```

Comment:
but how would they add timer events to the manager?  the manager could implement a `TimerFactory` interface (don't like that name):
```
type TimerFactory interface {
  New() timer  
}
```